src/style-sheets/pages/_page-not-found-view.scss

```diff
@@ -1,17 +1,86 @@
 .page-not-found-view-container {
-  position: relative;
   width: 100%;
-  height: 80vh;
-
-  .page-not-found-view-body {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    transform: translate(-50%, -50%);
-    text-align: center;
+  height: 100%;
+
+  & .page-not-found-view-body {
+    width: 83%;
+    height: 67%;
+    margin: 12.5% auto 21% auto;
+
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+
+    & .page-not-found-view-image-container {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      & img {
+        width: 100%;
+      }
+    }
+
+    & .page-not-found-view-text-container {
+      width: 51%;
+      margin: 5% auto 0;
+      text-align: center;
+
+      & p {
+        color: $olLightCopy;
+        font-size: $olH5Size;
```

Comment:
if we're using the size of an h5 should be just be using an h5 tag instead? I'd say if we were gonna control the size of a p tag in the module then we should be using rem for that.

Reply:
No, I don't think that should be a header.

Reply:
then if we're going to be changing the size in the module then we should be using rem instead of em right? https://css-tricks.com/rem-global-em-local/